docs/install/building.rst

```diff
@@ -27,3 +27,24 @@ available to turn on debug builds, address sanitizer, etc.).
 * :repo:`CMakeLists.txt`
 * :repo:`common.cmake`
 * :repo:`thirdparty.cmake`
+
+(Experimental) Building with Bazel_
```

Comment:
Yes I think it's a good idea to move this to the dev docs for now once that change gets merged.